lib/Conversion/HWToBTOR2/HWToBTOR2.cpp

```diff
@@ -156,15 +156,21 @@ struct ConvertHWToBTOR2Pass
   // If so, the original operation is returned
   // Otherwise the argument is returned as it is the original op
   Operation *getOpAlias(Operation *op) {
-    if (auto it = opAliasMap.find(op); it != opAliasMap.end())
+
+    // Remove the alias until none are left (for wires of wires of wires ...)
+    if (auto it = opAliasMap.find(op); it != opAliasMap.end()) {
       return it->second;
+    }
```

Comment:
Preferred LLVM style is to omit braces on single line bodys.

Reply:
oh yeah my bad, the braces were leftover from the while loop, I'll remove them.